src/renderer/k8s/api-base.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { apiBaseInjectionToken } from "../../common/k8s-api/api-base";
+import { JsonApi } from "../../common/k8s-api/json-api";
+import { apiPrefix, isDebugging } from "../../common/vars";
+import isDevelopmentInjectable from "../../common/vars/is-development.injectable";
+
+const apiBaseInjectable = getInjectable({
```

Comment:
Idea: everything related to `apiBase` strikes to me as overly complicated and could use some simpler competition somewhere else.

Reply:
Because it is essentially a wrapper of `fetch`?

Reply:
Something like that. Idea resolved anyway!